Source/Dafny/Plugins/PluginConfiguration.cs

```diff
@@ -22,10 +24,18 @@ public abstract class PluginConfiguration {
   }
 
   /// <summary>
-  /// Override this method to provide specific rewriters to Dafny
+  /// Override this method to provide rewriters
   /// </summary>
   /// <returns>a list of Rewriter that are going to be used in the resolution pipeline</returns>
   public virtual Rewriter[] GetRewriters(ErrorReporter errorReporter) {
     return Array.Empty<Rewriter>();
   }
+
+  /// <summary>
+  /// Override this method to provide specific compilers to Dafny
```

Comment:
```suggestion
  /// Override this method to provide compilers
```
To match the change you made on `GetRewriters`